ykcapi/src/lib.rs

```diff
@@ -153,9 +153,13 @@ pub extern "C" fn yk_stopgap(
     print_jit_state("enter-stopgap");
 
     // Parse AOTMap.
+    // FIXME: The memory behind this slice was malloced during compilation of the trace and thus
+    // must only be freed if the trace is too.
```

Comment:
You can implement freeing now in the `drop` of the trace, right? It just won't be called (since the trace isn't dropped).